packages/server/src/api/controllers/query/index.ts

```diff
@@ -1,15 +1,21 @@
 import { generateQueryID } from "../../../db/utils"
-import { BaseQueryVerbs, FieldTypes } from "../../../constants"
+import { BaseQueryVerbs } from "../../../constants"
 import { Thread, ThreadType } from "../../../threads"
 import { save as saveDatasource } from "../datasource"
 import { RestImporter } from "./import"
 import { invalidateDynamicVariables } from "../../../threads/utils"
 import env from "../../../environment"
-import { quotas } from "@budibase/pro"
 import { events, context, utils, constants } from "@budibase/backend-core"
 import sdk from "../../../sdk"
 import { QueryEvent } from "../../../threads/definitions"
-import { ConfigType, Query, UserCtx, SessionCookie } from "@budibase/types"
+import {
+  ConfigType,
+  Query,
+  UserCtx,
+  SessionCookie,
+  QuerySchema,
+  FieldType,
```

Comment:
Just a sanity check - what is the difference between 'FieldType' and 'FieldTypes' ?

Reply:
`FieldTypes` is the old non-TS way that we did this, it hasn't been fully cleaned up yet - I'll raise a separate PR to finally be done with it.

If an enumeration is plural, not singular then it is an old pre-TS type, which needs to be gotten rid of. It doesn't really impact anything, but just for consistency sake working on getting rid of them.